release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2396,6 +2396,116 @@ $ ./openshift-install wait-for install-complete
 
 * The `loadBalancerSourceRanges` attribute is not supported, and is thus ignored, in load-balancer type services in clusters that run on {rh-openstack} and use the OVN Octavia provider. There is no workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-2789[*OCPBUGS-2789*]) 
 
+* After a catalog source update, it takes time for OLM to update the subscription status.
+This can mean that if you modify a catalog source policy in a `ClusterGroupUpdate` CR, a subscription policy in the same `ClusterGroupUpdate` CR is not remediated as its status has not had sufficient time to update and continues to show it as compliant.
+As a workaround, include a status check in the catalog source policy in the `spec` section.
++
+[source,yaml]
+----
+metadata:
+  name: redhat-operators-disconnected
+spec:
+  displayName: disconnected-redhat-operators
+  image: registry.example.com:5000/disconnected-redhat-operators/disconnected-redhat-operator-index:v4.11
+status:
+  connectionState:
+    lastObservedState: READY
+----
++
+This mitigates the delay for OLM to pull the new index image and get the pod ready, reducing the overall time between completion of catalog source policy remediation and the update of the subscription status. 
+If the issue persists and the subscription policy status update is still late with the workaround, apply an identical `ClusterGroupUpdate` CR with a different name. 
+(link:https://issues.redhat.com/browse/OCPBUGS-2813[*OCPBUGS-2813*])
+
+* {cgu-operator} skips remediating a policy if all selected clusters are compliant when the `ClusterGroupUpdate` CR is started.
+The update of operators with a modified catalog source policy and a subscription policy in the same `ClusterGroupUpdate` CR does not complete. 
+The subscription policy is skipped as it is still compliant until the catalog source change is enforced.
+As a workaround, add a trivial change to one CR in the `common-subscription` policy, for example: 
```

Comment:
s/trivial change/the following

Reply:
Is this an example, or does the user always enter this string as shown? When?

Reply:
This is just an example. the idea is that the user has to make **a** change to the policy in order that it shows up as changed and loses its _compliant_ status. The kind of thing you might do in a PR if a check has failed for some reason other than an error in your branch. You'd (or at least I have in the past) make a change in the file just to have something to push again in order to trigger new checks and get the PR through.

Reply:
done